src/AdminConsole/Pages/App/Playground/NewAccount.cshtml

```diff
@@ -81,7 +81,7 @@
                                 </div>
                                 @if (!CurrentContext.Features.AllowAttestation)
                                 {
-                                    <warning-alert-box message="@((MarkupString)"Attestation is only available on 'Enterprise' plans, read more <a class=\"link-blue\" href=\"https://docs.passwordless.dev/guide/concepts.html#attestation\">here</a>.")"></warning-alert-box>
+                                    <warning-alert-box message="@((MarkupString)"Attestation is only available on 'Enterprise' plans, read more <a class=\"link-blue\" href=\"https://docs.passwordless.dev/guide/concepts.html#attestation\" target=\"_blank\">here</a>.")"></warning-alert-box>
```

Comment:
@jonashendrickx nitpick: I think you can remove the escaping if you use multiple `"""` to start the string: https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/raw-string